broker/import_broker.cpp

```diff
@@ -220,20 +220,20 @@ namespace RayGene3D
 
       if (node.rotation.size() == 4)
       {
-        const auto rotation = glm::quat(node.rotation[0], node.rotation[1], node.rotation[2], node.rotation[3]);
-        transform = glm::mat4_cast(rotation) * transform;
+        const auto rotation = glm::quat(node.rotation[3], node.rotation[0], node.rotation[1], node.rotation[2]);
+        transform = glm::mat4_cast(rotation);
       }
 
       if (node.translation.size() == 3)
       {
         const auto translation = glm::fvec3(node.translation[0], node.translation[1], node.translation[2]);
-        transform = glm::translate(transform, translation);
+        transform = glm::fmat4x4(transform[0], transform[1], transform[2], glm::fvec4(translation, 1.0f));
```

Comment:
Let us apply translation before rotation and invoke "translate" method instead of Ctor